pandas/tests/groupby/test_groupby.py

```diff
@@ -2174,3 +2174,14 @@ def test_groupby_series_with_tuple_name():
     expected = Series([2, 4], index=[1, 2], name=("a", "a"))
     expected.index.name = ("b", "b")
     tm.assert_series_equal(result, expected)
+
+
+def test_groupby_std_on_nullable_column():
+    # GH 35516
+    df = pd.DataFrame(
+        {"A": [2, 1, 1, 1, 2, 2, 1], "B": pd.Series(np.full(7, np.nan), dtype="Int64")}
+    )
+    result = df.groupby("A").std()
+    expected = DataFrame([np.nan, np.nan], index=[1, 2], columns=["B"])
```

Comment:
so actually i think there are some bugs here as std doesn't aggregate properly.

```
In [21]: df = pd.DataFrame(
    ...:         {"A": [2, 1, 1, 1, 2, 2, 1], "B": pd.Series(np.full(7, np.nan), dtype="Int64"), 'C': pd.array([1,2,3, 4,pd.NA, pd.NA, pd.NA], dtype='Int64')}
    ...:
    ...:         )

In [22]: df.groupby('A').std()
Out[22]:
    B    C
A
1 NaN  1.0
2 NaN  NaN

In [23]: df.groupby('A').sum()
Out[23]:
   B  C
A
1  0  9
2  0  1

In [24]: df.groupby('A').mean()
Out[24]:
      B    C
A
1  <NA>  3.0
2  <NA>  1.0
```

Reply:
But it seems its fine by now, no? The issue is about making a test to a problem that is apparently already solved

One thing that bothered me a bit while changing the tests to run on var() aswell is that it treats nan values a bit different? std() returns np.nan which creates a dtype float64, whereas var() returns pd.NA which creates a dtype Float64... But this is a separate topic right?

Reply:
the results are not correct

the issue is still broken - it's not just test only